src-ts/tools/learn/my-certificate/certificate/Certificate.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames'
+import { FC } from 'react'
+
+import { CertificateBgPattern } from './certificate-bg-pattern'
+import { CertificateTrackType } from './certificate-track-type.enum'
+import styles from './Certificate.module.scss'
+import { CourseCard } from './course-card'
+import { ReactComponent as TcAcademyLogoSvg } from './tc-academy-logo.svg'
+import { ReactComponent as TcLogoSvg } from './tc-logo.svg'
+import { ReactComponent as FccLogoSvg } from './vendor-fcc-logo.svg'
+
+interface CertificateProps {
+    completedDate?: string
+    course?: string
+    provider?: string
+    tcHandle?: string
+    userName?: string
+}
+
+const Certificate: FC<CertificateProps> = (props: CertificateProps) => {
+    const certificateType: CertificateTrackType = 'DEV'
+
+    return (
+        <div className={styles['wrap']}>
+            <div className={classNames(styles['details'], `theme-${certificateType.toLowerCase()}`)}>
+                <h2 className='details'>Topcoder Academy</h2>
+                <h3>Certificate of Course Completion</h3>
+                <h1 className={styles['username']}>
+                    {props.userName}
+                </h1>
+                <div className={classNames('large-subtitle', styles['tc-handle'])}>
+                    <span>topcoder handle: </span>
+                    <span>{props.tcHandle}</span>
+                </div>
+                <div className={styles['logos']}>
+                    <div className={styles['logo']}>
+                        <TcLogoSvg />
+                    </div>
+                    <div className={styles['divider']} />
+                    <div className={styles['logo']}>
+                        <TcAcademyLogoSvg />
+                    </div>
+                </div>
+            </div>
+            <div className={styles['badges']}>
+                <div className={styles['pattern-bg']}>
+                    <CertificateBgPattern type={certificateType} />
+                </div>
+                <div className={styles['course-card']}>
+                    <CourseCard
+                        type={certificateType}
+                        course={props.course}
```

Comment:
It's not clear to me from the Figma design if this should show the Course name "(New) Responsive Web Design" or the Certification name "Responsive Web Design". Since the certificate actually says "Certificate of Course Completion" I'm guessing it should just be the name of the course. @steveburnsTC What do you think?